back-end/services/confirmation-sender.ts

```diff
@@ -41,13 +42,16 @@ async function sendConfirmationToUser(order: Order) {
     const mailOptions = {
         from: `'Pizza Godfather' <${SMTP_EMAIL}>`,
         to: order.buyer.email,
-        subject: `New order - £${order.totalPayment} - ${order.buyer.firstName} ${order.buyer.lastName}`,
+        subject: `Your order was received`,
         html: storeEmailTemplate
     };
     return sendEmail(transporter, mailOptions);
 }
 
 function renderEjsTemplate(template: string, order: Order, isStoreCopy: boolean): Promise<string> {
+    order.orderItems.forEach(item => {
+        item.name = BasketService.getDescription(item);
```

Comment:
This is not a good idea because it will change the name on the domain entity (the order).

If another piece of code were to call save on that, it would overwrite the name of the item.

A good way to achieve this is to give EJS access to the BasketService.getDescription function. Simply add it to the view data, and then you can do this in EJS:
`<%= getDescription(item) %>`